internal/provider/credential_provider_resource_test.go

```diff
@@ -293,6 +293,42 @@ func TestAccCredentialProviderResource_OAuthClientCredentialsPostBody(t *testing
 	})
 }
 
+const testOAuthAuthorizationCodeResourceName string = "aembit_credential_provider.oauth_authorization_code"
+
+func TestAccCredentialProviderResource_OAuthAuthorizationCode(t *testing.T) {
+	createFile, _ := os.ReadFile("../../tests/credential/oauth-authorization-code/TestAccCredentialProviderResource.tf")
+	modifyFile, _ := os.ReadFile("../../tests/credential/oauth-authorization-code/TestAccCredentialProviderResource.tfmod")
```

Comment:
You're going to need to implement some config tweaking code here that replaces the id you started with (`a8f8ecdc-eb48-4a29-b854-d25745628b51`) with a random uuid.
Otherwise, the acceptance test will randomly fail whenever one of the 3 ci/cd runners happens to overlap.